apimodels/agent_models.go

```diff
@@ -120,6 +120,7 @@ type AgentSetupData struct {
 	TaskOutput             evergreen.S3Credentials `json:"task_output"`
 	EC2Keys                []evergreen.EC2Key      `json:"ec2_keys"`
 	TraceCollectorEndpoint string                  `json:"trace_collector_endpoint"`
+	ExecTimeoutLimit       int                     `json:"exec_timeout_limit"`
```

Comment:
nit: call it `MaxExecTimeoutSecs` would be preferable for consistency, also specifying that it's in seconds in the field name helps to know the units.

Reply:
updated